code/game/turfs/simulated/wall/reinf_walls.dm

```diff
@@ -15,9 +15,9 @@
 
 /turf/closed/wall/r_wall/examine(mob/user)
 	..()
-	deconstruction_hints_rwall(user)
+	deconstruction_hints(user)
```

Comment:
remove the examine, it's inherited.

Reply:
But the deconstruction hints don't show up if I remove the examine completely.